src/globals/formatting.ts

```diff
@@ -1,5 +1,7 @@
 export const formatNumber = function (value: number): string {
```

Comment:
why have a glocal formatting , its a vue filter for the cart right ?

Reply:
If you would have searched for the used instances, you'd see it is used in two places.